CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### 💡 Enhancements 💡
 
 - `k8seventsreceiver`: Add Api_version and resource_version (#8539)
+- `kafkaexporter`: Updated the README file to indicate supported pipeline types (#8814)
```

Comment:
No update to the changelog is needed for a documentation change. I've added the "Skip changelog" label.

```suggestion
```